From 4318b8ca1fe3f7b3df6864740cf3cb85be281fe6 Mon Sep 17 00:00:00 2001
From: Michal Marek <mmarek@suse.cz>
Date: Tue, 10 Jun 2014 17:18:56 +0200
Subject: [PATCH 1/3] kabi: Lift ifdefs for some disabled network protocols
Patch-mainline: Never, kabi
References: fate#312013

This allows the modules to be built out-of-tree, if needed.

Signed-off-by: Michal Marek <mmarek@suse.cz>
---
 include/net/net_namespace.h | 2 +-
 net/core/secure_seq.c       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/net/net_namespace.h b/include/net/net_namespace.h
index 9d22f08..a45547b 100644
--- a/include/net/net_namespace.h
+++ b/include/net/net_namespace.h
@@ -92,7 +92,7 @@ struct net {
 #if defined(CONFIG_IP_SCTP) || defined(CONFIG_IP_SCTP_MODULE)
 	struct netns_sctp	sctp;
 #endif
-#if defined(CONFIG_IP_DCCP) || defined(CONFIG_IP_DCCP_MODULE)
+#if defined(CONFIG_IP_DCCP) || defined(CONFIG_IP_DCCP_MODULE) || 1
 	struct netns_dccp	dccp;
 #endif
 #ifdef CONFIG_NETFILTER
diff --git a/net/core/secure_seq.c b/net/core/secure_seq.c
index 8d9d05e..3d3c32d 100644
--- a/net/core/secure_seq.c
+++ b/net/core/secure_seq.c
@@ -154,7 +154,7 @@ u32 secure_ipv4_port_ephemeral(__be32 saddr, __be32 daddr, __be16 dport)
 EXPORT_SYMBOL_GPL(secure_ipv4_port_ephemeral);
 #endif
 
-#if IS_ENABLED(CONFIG_IP_DCCP)
+#if IS_ENABLED(CONFIG_IP_DCCP) || 1
 u64 secure_dccp_sequence_number(__be32 saddr, __be32 daddr,
 				__be16 sport, __be16 dport)
 {
-- 
1.8.5.2

